api-reference/beta/api/device_post_registeredusers.md

```diff
@@ -32,33 +31,20 @@ Here is an example of the request.
   "name": "create_directoryobject_from_device"
 }-->
 ```http
-POST https://graph.microsoft.com/beta/devices/{id}/registeredUsers
+POST https://graph.microsoft.com/beta/devices/{id}/registeredUsers/$ref
 Content-type: application/json
-Content-length: 30
+Content-length: 97
 
 {
-  "directoryObject": {
-  }
+  "@odata.id":"https://graph.microsoft.com/beta/users/af9f55cc-78ad-4476-be6e-f9bb86a3dddd"
 }
 ```
 In the request body, supply a JSON representation of [directoryObject](../resources/directoryobject.md) object.
```

Comment:
One other thing -- we are citing  "user" in the Request Body section and "directoryObject" in the Example request (and actually referencing a user).